src/main/kotlin/no/nav/familie/tilbake/behandling/steg/StegService.kt

```diff
@@ -54,6 +54,10 @@ class StegService(
             )
         }
 
+        if (behandledeSteg.behandlingsstatus != behandling.status){
```

Comment:
Genialt hvis vi kan gjøre det så enkelt.

Men er usikker på om dette vil virke. kan vi ikke ha behandling som utredes mens steget er opprettet?

Vi bør isåfall se om vi har gode nok tester rundt dette :)